ts/IterableInboxMessageDisplay.tsx

```diff
@@ -46,13 +50,59 @@ const IterableInboxMessageDisplay = ({
    returnButtonContainer = {...returnButtonContainer, marginTop: Platform.OS === 'android' ? 0 : 40}
    returnButtonContainer = (!isPortrait) ? {...returnButtonContainer, marginTop: 10} : returnButtonContainer
 
+   let JS = `
+      const links = document.querySelectorAll('a')
+      links.forEach(link => {
+         if(link.href === "iterable://dismiss") {
+            link.class = "dismiss"   
+         }
+
+         if(link.href === "iterable://delete") {
+            link.class = "delete"
+         }
+
+         link.addEventListener("click", () => {
+            if(link.class === "dismiss") {
+               window.ReactNativeWebView.postMessage("iterable://dismiss")
+            } else if(link.class === "delete") {
+               window.ReactNativeWebView.postMessage("iterable://delete")
+            } else {
+               window.ReactNativeWebView.postMessage(link.href)
+            }
+         })
+
+         if(link.href === "iterable://dismiss" || link.href === "iterable://delete") {
+            link.href = "javascript:void(0)"
```

Comment:
This is to avoid accidental multiple link clicks, right? Or is there usage too?

Reply:
As addressed above, changing the href of the link prevents the HTML from trying to open the iterable action.